src/components/settings/Account/Billing/alerts/Alert.tsx

```diff
@@ -0,0 +1,384 @@
+import {
+  AccountStatus,
+  useAccount,
+  useAccountUsage,
+} from "@3rdweb-sdk/react/hooks/useApi";
+import { useLoggedInUser } from "@3rdweb-sdk/react/hooks/useLoggedInUser";
+import { useTrack } from "hooks/analytics/useTrack";
+import { useLocalStorage } from "hooks/useLocalStorage";
+import { useCallback, useMemo, useState } from "react";
+import { useRouter } from "next/router";
+import {
+  useDisclosure,
+  Alert,
+  Flex,
+  AlertIcon,
+  AlertTitle,
+  AlertDescription,
+  IconButton,
+  Stack,
+} from "@chakra-ui/react";
+import { OnboardingBilling } from "components/onboarding/Billing";
+import { OnboardingModal } from "components/onboarding/Modal";
+import { FiX } from "react-icons/fi";
+import { Text, Heading, TrackedLinkButton } from "tw-components";
+import { ManageBillingButton } from "../ManageButton";
+import { RecurringPaymentFailureAlert } from "./RecurringPaymentFailureAlert";
+import { format } from "date-fns";
+import { ServiceCutoffAlert } from "./ServiceCutoffAlert";
+
+type AlertConditionType = {
+  shouldShowAlert: boolean;
+  key: string;
+  title: string;
+  description: string;
+  status: "error" | "warning";
+  componentType:
+    | "serviceCutoff"
+    | "recurringPayment"
+    | "paymentVerification"
+    | "usage";
+};
+
+export const BillingAlerts = () => {
+  const { isLoggedIn } = useLoggedInUser();
+  const usageQuery = useAccountUsage();
+  const meQuery = useAccount({ refetchInterval: 1000 });
```

Comment:
This will poll every 1000ms forever. The parameter also takes in a function that allows you to specify a condition to stop polling

Reply:
yes this one lets please fix

Reply:
Done ✅